RotatedArray.java

```diff
@@ -0,0 +1,38 @@
+// Time Complexity :O(logn)
+// Space Complexity :O(1)
+// Did this code successfully run on Leetcode :yes
+// Any problem you faced while coding this :Yes took very long to come up with this. had to take external help
+
+
+// Your code here along with comments explaining your approach
+
+class Solution {
+    public int search(int[] nums, int target) {
+        int left=0;
+        int right=nums.length-1;
+        
+        while(left<=right){
+            int mid = (left+right)/2;
```

Comment:
Sometimes this might lead to integer overflow, practice using (left + (right-left)/2) option